### ISSUE TITLE
Merge Alacrity armor and speed armor

### ISSUE THREAD

**Issue by DiamondWalker** [Tslat/Advent-Of-Ascension#512]
Alacrity and speed armor are pretty much useless. How many people have you seen using one of those two armor sets? Probably not too many. How many people have you seen using the upgrades (phantasmal/lunar amor)? Probably a lot more. It's because both speed armor and alacrity armor require sapphire, making them insanely expensive (I've never found a single sapphire in all my time playing AoA, let alone 12). While the upgrades are more late-game, they are much less grindy, so people usually go straight to them.

I think the Speed Armor should be removed, and its set bonus should be added to Alacrity armor. During my JitL days, I learned a mix between speed and jump height is incredibly useful, and would definitely be worth the grind. I would also give the equipped speed armor texture to the ice armor, since it's exactly the same sprite, but speed armor looks slightly more polished.

**Comment by Cruuk** [comment]
Alacrity useless? Holy fuck have you seen it?

Ok sorry for being abrasive, but Alacrity armor has 21 armor, compared to Diamond's 20. Even if it's still "earlygame" it is still one of the best sets in the game. Jump boost + fall damage resistance goes a looooong way. It does not need speed at all. Already it's really reliable to have throughout the whole game, no other armor set other than Lunar replicates the reliability of Alacrity in terms of travel (Rockbone has fall damage resist but is still very different, Hunter still has low grav/speed but only increases your jump by a tiny bit now)

Also Lunar is more of a sidegrade now, it has 20 armor with the only additional effect being night vision, the healing on jump thing was removed.

Also by never finding Sapphire, are you referring to it's current state in 3.0.2 or in general? If it's the former, that's getting fixed.
If it's the latter, you're just unlucky dude. Sapphire really isn't extremely hard to find if you just know what you're doing, like chunk-mining. I've been able to find enough Sapphire to make Alacrity a few times in my time playing AoA in survival.

Alacrity is completely fine as it is. Giving an earlygame set that much power with Speed armor's speed 2 and Alacrity's jump boost + fall damage resist would be busted.

I do understand your gripes with Speed armor though, not really worth going after considering the cost and Phantasm is a much much cheaper sidegrade.
